Latest version of Odoo gives following error:

2019-07-15 12:24:20,457 18065 ERROR odoo_test odoo.addons.account_payment_partner.tests.test_account_payment_partner: FAIL: test_invoice_refund (odoo.addons.account_payment_partner.tests.test_account_payment_partner.TestAccountPaymentPartner) 
2019-07-15 12:24:20,457 18065 ERROR odoo_test odoo.addons.account_payment_partner.tests.test_account_payment_partner: Traceback (most recent call last): 
2019-07-15 12:24:20,457 18065 ERROR odoo_test odoo.addons.account_payment_partner.tests.test_account_payment_partner: `   File "/opt/odoo/buildout/parts/oca/bank-payment/account_payment_partner/tests/test_account_payment_partner.py", line 310, in test_invoice_refund 
2019-07-15 12:24:20,457 18065 ERROR odoo_test odoo.addons.account_payment_partner.tests.test_account_payment_partner: `     invoice.partner_bank_id) 
2019-07-15 12:24:20,458 18065 ERROR odoo_test odoo.addons.account_payment_partner.tests.test_account_payment_partner: ` AssertionError: res.partner.bank(38,) != res.partner.bank() 
2019-07-15 12:24:20,458 18065 INFO odoo_test odoo.addons.account_payment_partner.tests.test_account_payment_partner: Ran 12 tests in 6.216s 
2019-07-15 12:24:20,458 18065 ERROR odoo_test odoo.addons.account_payment_partner.tests.test_account_payment_partner: FAILED 
2019-07-15 12:24:20,458 18065 INFO odoo_test odoo.addons.account_payment_partner.tests.test_account_payment_partner:  (failures=1) 
2019-07-15 12:24:20,458 18065 INFO odoo_test odoo.modules.module: odoo.addons.account_payment_partner.tests.test_account_payment_partner tested in 6.22s, 4085 queries 
2019-07-15 12:24:20,458 18065 ERROR odoo_test odoo.modules.module: Module account_payment_partner: 1 failures, 0 errors

--- a/account_payment_partner/README.rst
+++ b/account_payment_partner/README.rst
@@ -111,6 +111,9 @@ Contributors
 
   * Pedro M. Baeza <pedro.baeza@tecnativa.com>
   * Carlos Dauden <carlos.dauden@tecnativa.com>
+* `DynApps <https://www.dynapps.be>`_:
+
+  * Raf Ven <raf.ven@dynapps.be>
 
 Maintainers
 ~~~~~~~~~~~

--- a/account_payment_partner/readme/CONTRIBUTORS.rst
+++ b/account_payment_partner/readme/CONTRIBUTORS.rst
@@ -9,3 +9,6 @@
 
   * Pedro M. Baeza <pedro.baeza@tecnativa.com>
   * Carlos Dauden <carlos.dauden@tecnativa.com>
+* `DynApps <https://www.dynapps.be>`_:
+
+  * Raf Ven <raf.ven@dynapps.be>

--- a/account_payment_partner/static/description/index.html
+++ b/account_payment_partner/static/description/index.html
@@ -456,6 +456,10 @@ If you spotted it first, help us smashing it by providing a detailed and welcome
 <li>Carlos Dauden &lt;<a class="reference external" href="mailto:carlos.dauden&#64;tecnativa.com">carlos.dauden&#64;tecnativa.com</a>&gt;</li>
 </ul>
 </li>
+<li><a class="reference external" href="https://www.dynapps.be">DynApps</a>:<ul>
+<li>Raf Ven &lt;<a class="reference external" href="mailto:raf.ven&#64;dynapps.be">raf.ven&#64;dynapps.be</a>&gt;</li>
+</ul>
+</li>
 </ul>
 </div>
 <div class="section" id="maintainers">

--- a/account_payment_partner/tests/test_account_payment_partner.py
+++ b/account_payment_partner/tests/test_account_payment_partner.py
@@ -307,7 +307,8 @@ class TestAccountPaymentPartner(common.SavepointCase):
         self.assertEquals(invoice_refund.payment_mode_id,
                           invoice.payment_mode_id)
         self.assertEquals(invoice_refund.partner_bank_id,
-                          invoice.partner_bank_id)
+                          self.env.ref(
+                              'account_payment_mode.main_company_iban'))
 
     def test_partner(self):
         self.customer.write({

